src/components/SignerPanel/SignerPanel.js

```diff
@@ -458,12 +458,16 @@ export default function(props) {
     setActivityNonce,
   } = useContext(ActivityContext)
 
+  if (!wallet.web3) {
```

Comment:
Maybe we can show an error panel instead? Or should this never happen?

Reply:
Fixed!